### PR TITLE
Current hotkey settings are respected during the round + refactor to hotkey switching

### LINKED ISSUE #15556: Hotkey mode still doesn't stay the same when you switch mobs
Althought the annoying issue of looking like it did is out, it still doesn't keep the preference when you enter a new mob.

### PR DESCRIPTION
**Pull request by X-TheDark**
**TL;DR** Your current hotkey setting is remembered when you Ghost/Switch Mobs/etc. Also moves client verbs from `game/verbs` to `modules/client/verbs`

fixes #15556

**Functionality changes**
- When client connects, your preferences are used to set your hotkey mode;
- When you Ghost/Switch Mobs/etc, your current hotkey mode is used, rather than preferences;

**Refactors**
- Hidden 'Set Hotkeys' verb to set the mob-specific hotkeys (Borgs use different hotkeys);
- Moves client procs to 'modules/client/verbs'

:cl: X-TheDark
add: Switching mobs/ghosting/etc related actions during the round will now no longer reset your hotkey settings.
/:cl:

### CODE REVIEW

**Comment by Man-Shroom-** [comment]
you can't use tab to toggle hotkeys anymore?

that sux

**Comment by X-TheDark** [comment]
No, no, you still can. I just added new way to do it (you get a new verb). Tab still works as it did.

**Comment by MrStonedOne** [comment]
>it's functionality is unchanged

Not true, by making it a server side verb now, there is a delay before it changes.

Please fix kthx

(make it do both server side shit and non-server side shit, or! just get the status using winget rather than trying to track it)

**Comment by MrStonedOne** [comment]
Also, please refer to any issues this fix with `fixes #number` in the pr description so that github will auto close the issue when the pr is merged.

**Comment by X-TheDark** [comment]
In this commit:
* Reverts TAB changes (didn't know about responsiveness issues) - back to using hardcoded commands.
* Removes 'hotkeys' var from client defines (as I use `winget()` now, as MSO suggested);
* Reverts all the stupid hotkey name changes that I did (I did this to differentiate variable names between preferences and your current settings);
* Renames the verb proc to `sethotkeys()` and the name to 'Set Hotkeys'. Makes it hidden as calling it in-game (since it has a parameter) is complete ass;

**Comment by optimumtact** [comment]
Oh I didn't see your last comment

if you want to make it fully a proc go right ahead.
